### PR TITLE
Langevin docs update

### PR DESCRIPTION
**Pull request by crisbutu**
## Description

Made two updates to the documentation related to rotational motion with Langevin dynamics:
1. redefined L as angular velocity instead of angular momentum
2. changed the units of gamma_r from time^-1 to mass * length^2 * time^(-1)

## Motivation and context

Resolves #1768 

## How has this been tested?

Manually made sure the latex changes parsed correctly by pasting them into an online latex editor.

## Change log

```

```

## Checklist:

- [X] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/CONTRIBUTING.rst).
- [X] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/ContributorAgreement.md).
- [X] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

File: hoomd/md/methods/methods.py [hunk 1]
```diff
@@ -753,7 +753,7 @@ class Langevin(Method):
 
         gamma_r (TypeParameter[``particle type``,[`float`, `float` , `float`]]):
             The rotational drag coefficient tensor for each particle type
-            :math:`[\mathrm{time}^{-1}]`.
+            :math:`[\mathrm{mass} \cdot \mathrm{length}^{2} \cdot \mathrm{time}^{-1}]`.
```
Comment (tommy-waltmann) [inline hoomd/md/methods/methods.py]: This change is good, make sure to also change the units in the parameter description at the top of the docstring for `default_gamma_r`.

File: hoomd/md/methods/methods.py [hunk 2]
```diff
@@ -691,7 +691,7 @@ class Langevin(Method):
     where :math:`\vec{\tau}_\mathrm{C} = \vec{\tau}_\mathrm{net}`,
     :math:`\gamma_r^i` is the i-th component of the rotational drag coefficient
     (`gamma_r`), :math:`\tau_\mathrm{R}^i` is a component of the uniform random
-    the torque, :math:`\vec{L}` is the particle's angular momentum and :math:`I`
+    the torque, :math:`\vec{L}` is the particle's angular velocity and :math:`I`
```
Comment (tommy-waltmann) [inline hoomd/md/methods/methods.py]:
```suggestion
    torque, :math:`\vec{L}` is the particle's angular velocity and :math:`I`
```
Not related to units or variables, but I think there is an extra "the" in that sentence.

**Review by tommy-waltmann — CHANGES_REQUESTED** [review]
Thanks for the PR, we always like it when external folks contribute fixes!

One more thing for discussion: if $\vec L$ is actually the angular velocity, should we change the variable in the docstring to be $\vec \omega$ so its consistent with standard variable naming conventions?

**Comment by crisbutu** [comment]
@tommy-waltmann I addressed the comments and this is ready for another look!

**Comment by tommy-waltmann** [comment]
pre-commit.ci autofix

**Comment by tommy-waltmann** [comment]
This looks good, we now just need to satisfy the code and documentation style enforced by `pre-commit`. A list of lines that need fixing is included here: https://results.pre-commit.ci/run/github/147663007/1714140577.QTa6j2IcTPmDJO5n3xovrg .

**Review by AlainKadar — APPROVED** [review]

**Comment by joaander** [comment]
You can get fixes for:
```
hoomd/simulation.py:560:45: E226 missing whitespace around arithmetic operator
sphinx-doc/conf.py:63:21: E201 whitespace after '{'
sphinx-doc/conf.py:63:26: E202 whitespace before '}'
```
by merging the upstream `trunk-patch` into your branch.

**Comment by crisbutu** [comment]
@tommy-waltmann `pre-commit` now passes

**Review by tommy-waltmann — APPROVED** [review]
Thanks for the fix!

**Review by SchoeniPhlippsn — APPROVED** [review]

**Pull request merged**